verifiable-db/src/revelation/revelation_without_results_tree.rs

```diff
@@ -71,6 +80,46 @@ pub struct RevelationWithoutResultsTreeCircuit<
     pub(crate) check_placeholder: CheckPlaceholderGadget<PH, PP>,
 }
 
+/// Data structure containing the wires corresponding to public inputs
+/// of the query proof employed in the revelation circuit. It is a
+/// data structure employed to represent the public inputs of the query
+/// proof in a revelation circuit both for query proofs generated with
+/// batching circuits, and for query proofs generated with non-batching
+/// circuits
+pub(crate) struct QueryProofInputWires<const S: usize>
```

Comment:
Could we put that into its own `public_inputs.rs` file then ?

Reply:
They are not a new set of public inputs, they are just a subset of other public input sets. I would likely remove this struct once I will remove the old query circuits in a further PR, so I think it's not worthy to work further on them. Wdyt?